specifications/booking.yaml

```diff
@@ -40,7 +44,7 @@ paths:
       tags:
         - Listing
       parameters:
-        - name: from
+        - name: _from
```

Comment:
Lets see if there is a way to keep this implementation detail from leaking into the abstraction.

Reply:
Didn't change it because the code won't work without that change. I agree that this is a workaround and I am not happy with it but we should proceed until we have a better solution.